repos/fdbt-site/src/layout/SubNavigation.tsx

```diff
@@ -28,19 +28,19 @@ const SubNavigation = (): ReactElement => {
                         </a>
                     </li>
 
-                    <li className="app-subnav__section-item">
+                    {/*<li className="app-subnav__section-item">
```

Comment:
Is this to be deleted?

Reply:
They'll be added in again as we add the other global settings, I did toy with the idea of deleting them as they will be easy to re-add. What do you think?